.vscode/settings.json

```diff
@@ -6,5 +6,6 @@
 		"out": true // set this to false to include "out" folder in search results
 	},
 	"npm.exclude": ["**/src"],
-	"npm.enableScriptExplorer": true
+	"npm.enableScriptExplorer": true,
+	"typescript.tsdk": "node_modules\\typescript\\lib"

```

Comment:
This is so when developing, vscode will use the typescript version of the project and not the global typescript.